### PR TITLE
add android logcat settings docs

### PR DESCRIPTION
**Pull request by anjalin14**
### Purpose of PR
**Type of change:**
- [ ] Documentation update for Android Logcat Settings window

* **Description of the feature**
_What it does, why was it created, how does it work and so on..._
* **Links to screenshots, design docs, user docs, etc.**
* **Links to Jira/Fogbugz cases**
* **Halo effect**
_What areas do you know/expect this feature to affect? Do you have any worries about possible breaking points? Write them down here._
* **Performance Impact**
_Could this PR affect performance? If so, what has been done to measure time taken per frame, memory used, etc.?_
* **Package requirements**
_What Unity Editor version is required/supported? Any other dependancy packages needed?_

### Checklist for PR maker
- [ ] Have you added a backport label? (if needed)
- [ ] Have you updated the Changelog? _Each package has a `CHANGELOG.md` file_
- [ ] **Have you added or updated the Documentation to your PR?** _When you add a new feature, change a property name, or change the behaviour of a feature, it's best practice to include related documentation changes in the same PR_
---
### Testing status
* **Existing or new automation tests - what automation was added, changed**
_Put any relevant Yamato links here_
* **Detailed description of what testing was done, what projects were run**
_Tell the reviewers what did you already test, so that they may have a better understanding when reviewing/testing this PR themselves_
* **What environment did you test on?**
_Your Operating System, Platforms you ran on, Graphics APIs and any other relevant info you can think of_
* **Projects**
_Any existing projects ready for testing should be noted here_
* **Testing checklist**
- [ ] Built and run editor Locally or Katana
- [ ] Built a player Android/iOS (if applicable)
- [ ] Run on device Android/iOS (if applicable)
- [ ] If new feature has UI, does it match Unity style? (Unity [HIG](https://unitytech.github.io/unityeditor-hig/)?)
- [ ] Tested Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] All items have tooltips?
---
### Note to guardian
_Changeset to use, if extra changesets on top. Any other notes to pass along_

### Comments to reviewers
_Notes for the reviewers you have assigned_

### CODE REVIEW

File: com.unity.mobile.android-logcat/Documentation~/TableOfContents.md [hunk 1]
```diff
@@ -21,3 +21,4 @@
     * [Memory window](memory-window.md)
       * [Memory window reference](memory-window-reference.md)
     * [Input window](inputs-window.md)
+* [Android Logcat Settings](android-logcat-settings)
```
Comment (todi1856) [inline com.unity.mobile.android-logcat/Documentation~/TableOfContents.md:24]: Shouldn't it be (android-logcat-settings.md)

File: com.unity.mobile.android-logcat/Documentation~/android-logcat-settings.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  - [Stacktrace Regex](#stacktrace-regex)
+  
+**Note**: Use the **Reset** button at the bottom of the Settings window to restore the default settings.
+
+## Messages
+
+Use **Messages** section to customize the appearance and the number of the messages displayed on the Android Logcat window.
+
+![](images/logcat-settings-messages.png)
+> Messages
```
Comment (todi1856) [inline com.unity.mobile.android-logcat/Documentation~/android-logcat-settings.md:20]: what does this tag do ? It looks a bit weird when looking via https://github.com/Unity-Technologies/com.unity.mobile.logcat/blob/docr-3561/com.unity.mobile.android-logcat/Documentation~/android-logcat-settings.md in github website

**Review by todi1856 — CHANGES_REQUESTED** [review]
Few questions

File: com.unity.mobile.android-logcat/Documentation~/TableOfContents.md (hunk 1 above)
Comment (anjalin14) [reply com.unity.mobile.android-logcat/Documentation~/TableOfContents.md:24]: yes

File: com.unity.mobile.android-logcat/Documentation~/android-logcat-settings.md (hunk 2 above)
Comment (anjalin14) [reply com.unity.mobile.android-logcat/Documentation~/android-logcat-settings.md:20]: It is for displaying the name of the image file. I added it after comparing with the existing ones.

**Review by todi1856 — APPROVED** [review]

File: com.unity.mobile.android-logcat/Documentation~/android-logcat-settings.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Message Colors
+
+![](images/logcat-settings-message-colors.png)
+
+
+Use **Message Colors** setting to customize the color of the log messages displayed in the Android Logcat window. Customizing the colors can help you differentiate between different types of log messages, making it easier for you to identify any issues. You can assign different colors to log messages based on their priorities, such as **Verbose**, **Debug**, **Info**, **Warn**, **Error**, and **Fatal**. For more information about the priority levels, refer to Android documentation on [Filtering log output](https://developer.android.com/tools/logcat#filteringOutput).
+
+To customize the log message color, click anywhere on the swatch displayed next to the log message priority level, which opens the color picker. On the
+the color picker interface, you can use slider or text box to set your preferred RGB or Hexadecimal color code. Alternatively, you can set the color you prefer using the **Swatches** option at the bottom of the color picker. For more information about Swatches, refer to <a href="https://docs.unity3d.com/Manual/EditingValueProperties.html#swatch-libraries">Creating swatch libraries</a>.
```
Comment (kilroyandrew) [inline com.unity.mobile.android-logcat/Documentation~/android-logcat-settings.md:37]: Just to flag there's a double `the` at the end of line 36 and start of 37.

**Pull request merged**